dashboard/app/views/levels/_dialog.html.haml

```diff
@@ -72,6 +72,19 @@
         %button#ok-button.btn-danger{style: 'float: right;'}
           =t('dialog.startover_ok')
 
+  #instructions-dialogcontent
+    .modal-content.no-modal-icon.markdown-instructions-container
+      %p.dialog-title
+        ="#{t('puzzle')} #{@script_level.try(:position) || 1} #{t('of')} #{@script_level.try(:stage_total) || 1}"
+      .instructions-markdown
+        %p
+          -# CSP is en_us only, always show markdown instructions
```

Comment:
`_dialog.html.haml` is not, but `_widget.html.haml` which consumes it is only for CSP Widgets.  It's not ideal to include this extra markup on every DSLDefined page, but we're already doing this for 5 other dialogs.  When these dialogs are refactored they can be moved to JS all at once.